app/model/orga.py

```diff
@@ -0,0 +1,79 @@
+import enum
+from sqlalchemy_utils import generic_relationship, ChoiceType
+
+from model.abc import db, Base
+
+
+class OrganisationType(enum.IntEnum):
+    finess_et = 1
+    company = 2
+
+    @classmethod
+    def get_value(cls, name):
+        return getattr(cls, name).value
+
+    @classmethod
+    def get_name(cls, value):
+        return cls(value).name
+
+    def __int__(self):
+        return self.value
+
+
+class Platform(Base):
```

Comment:
In the future, we might want to add contacts information for the Platform.

Reply:
When we need contact, we add it.